docs/developer-guide/ksqldb-clients/java-client.md

```diff
@@ -643,6 +644,30 @@ for (QueryInfo query : queries) {
 }
 ```
 
+See the [API reference](https://docs.ksqldb.io/en/latest/developer-guide/ksqldb-clients/java-client/api/io/confluent/ksql/api/client/Client.html)
+for more information.  
+
+Describe specific streams and tables<a name="describe-source"></a>
+------------------------------------------------------------------------
+
+Starting with ksqlDB 0.12.0, the `describeSource()` method allows client apps
```

Comment:
```suggestion
Starting with ksqlDB 0.12.0, the `describeSource()` method enables client apps
```

Reply:
I always forget which one is preferred 😆 Will make the change.

Reply:
Usually, we reserve "allow" for security-related stuff, like "allow traffic through the firewall".